apps/src/gamelab/GameLabSprite.js

```diff
@@ -14,6 +14,9 @@ module.exports.createSprite = function (x, y, width, height) {
   var s = new this.Sprite(x, y, width, height);
   var p5Inst = this;
 
+  s._horizontalStretch = 1;
+  s._verticalStretch = 1;
```

Comment:
Since this basically the initial definition for these, it'd be nice to give them a doc comment explaining their (somewhat unusual) purpose and a `@type {number}` tag.